CMakeLists.txt

```diff
@@ -626,8 +626,8 @@ if (NOT MINGW AND NOT MSVC AND NOT APPLE)
     configure_file(build/Linux+BSD/mscore.desktop.in mscore${MSCORE_INSTALL_SUFFIX}.desktop)
     install( FILES ${PROJECT_BINARY_DIR}/mscore${MSCORE_INSTALL_SUFFIX}.desktop DESTINATION share/applications)
     # Install appdata file (perform variable substitution first)
-    configure_file(build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in org.musescore.MuseScore.appdata.xml)
-    install( FILES ${PROJECT_BINARY_DIR}/org.musescore.MuseScore.appdata.xml DESTINATION share/appdata)
+    configure_file(build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in org.musescore.MuseScore${MSCORE_INSTALL_SUFFIX}.appdata.xml)
+    install( FILES ${PROJECT_BINARY_DIR}/org.musescore.MuseScore.appdata.xml DESTINATION share/metainfo)
```

Comment:
wrong filename (I’ll provide a list of fixes below)

Reply:
What’s wrong? It is supposed to be R-DNS format as per Appstream standards, I just added the suffix as you suggested.

Reply:
You forgot the second occurrence of the filename in the install line (see the commits of mine below).

Reply:
Fixed.